internal/backends/python/gen_pypi_map/package_index.go

```diff
@@ -26,7 +26,13 @@ func FakePackageIndex(packages ...string) PackageIndex {
 }
 
 func NewPackageIndex(index string, limit int) (PackageIndex, error) {
-	resp, err := http.Get(index)
+	req, err := http.NewRequest("GET", index, nil)
+	if req != nil {
+		return PackageIndex{}, err
+	}
+
+	req.Header.Add("User-Agent", "upm (+https://github.com/replit/upm)")
+	resp, err := http.DefaultClient.Do(req)
```

Comment:
```suggestion
	resp, err := api.HttpClient.Get(index)
```
Any reason we can't do this?

Reply:
i don't believe so! i'll try it